### PR TITLE
fix: exception when using hideSidebar

### LINKED ISSUE #1392: Hidden sidebar causing other plugins issues
#### Steps to reproduce

1. Setup fresh site
1. Add config:
    ```js
    window.$docsify = {
      name: '',
      repo: '',

      // cover page doc
      coverpage: true,
      homepage: 'home.md',

      // remove sidebar
      hideSidebar: true,

      // auto scroll to top
      auto2top: true
    }
    ```
1. Create two or more `md` pages:
    1. `home.md`  -- add 2x `href` to  `page1` and `page2`
    1. `page1.md`  -- add 2x `href` to  `home` and `page2`
    1. `page2.md`  -- add 2x `href` to  `home` and `page1`
1. Launch http://localhost:3000
1. From `/#/home` in console no errors
    1. Click on `page1` link
    1. Continue to cycle through pages without hard refresh

#### What is current behaviour

After first load and the sidebar is removed, any other navigation to other pages triggers the `.remove` which fails since the element is already gone.

If the error is hit, then following plugins dont run until a hard reload.

```js
Uncaught TypeError: Cannot read property 'remove' of null
    at Vn.Xn._renderSidebar (docsify.min.js:1)
    at Vn.Pn (docsify.min.js:1)
    at docsify.min.js:1
    at o (docsify.min.js:1)
    at B (docsify.min.js:1)
    at t (docsify.min.js:1)
    at docsify.min.js:1
    at docsify.min.js:1
    at docsify.min.js:1
    at On (docsify.min.js:1)
```

```js
Line 4150: return document.querySelector("aside.sidebar").remove(),
```

#### What is the expected behaviour

Error doesn't occur, and doesn't stop other scripts from running.

#### Other relevant information

- [x] Bug does still occur when all/other plugins are disabled?

- Your OS: macOS
- Node.js version: 14.8.0
- npm/yarn version: 6.14.8
- Browser version: 
- Docsify version: 4.11.6
- Docsify plugins:

### PR DESCRIPTION
**Pull request by sy-records**
**Summary**

fix #1392

If there is a better solution, you can also submit other PR.

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Repo settings
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

### CODE REVIEW

**Review by anikethsaha — CHANGES_REQUESTED** [review]
I think this will result in inconsistent styling if suppose the first line runs successfully but there is an error in second-line then `aside..sidebar` will be removed already but not the rest of the styling as it would return null.  (let me know if I am missing anything)

How about checking all these in one `if` statement 

like this

```js

if( document.querySelector('aside.sidebar') &&  document.querySelector('button.sidebar-toggle') && ...){
 document.querySelector('aside.sidebar').remove(); 
 document.querySelector('button.sidebar-toggle').remove(); 
...
}
```


What do you think ?

**Comment by sy-records** [comment]
I think it's good, we just need to judge `sidebar` and `sidebar-toggle`.

**Comment by anikethsaha** [comment]
Cool, You can make changes in this PR itself

File: src/core/render/index.js [hunk 1]
```diff
@@ -132,8 +132,13 @@ export function renderMixin(proto) {
 
     if (hideSidebar) {
       // FIXME : better styling solution
```
Comment (anikethsaha) [inline src/core/render/index.js:134]: LGTM

lets remove this comment for now

**Review by anikethsaha — APPROVED** [review]
LGTM

one small suggestion

File: src/core/render/index.js (hunk 1 above)
Comment (sy-records) [reply src/core/render/index.js]: done.

**Review by anikethsaha — APPROVED** [review]

**Pull request closed without merging**